superset-frontend/src/SqlLab/components/QueryTable.jsx

```diff
@@ -98,6 +130,36 @@ const QueryTable = props => {
           q.duration = fDuration(q.startDttm, q.endDttm);
         }
         const time = moment(q.startDttm).format().split('T');
+        const statusConfig = {
```

Comment:
i'd move this into a function where we pass `state` and returns the updated `statusConfig`
```
const updateStatusConfig = (state) => {
 //...
}
```